docker/Dockerfile.debug

```diff
@@ -1,9 +1,5 @@
 FROM ubuntu:xenial
 # Do not add more stuff to this list that isn't small or critically useful.
-# If you occasionally need something on the container do
```

Comment:
this is quite useful information, the idea here is not to remove the comment but update it or make sure it still works with the new iptable script or replace with a pointer to new instructions (or maybe move it to the pilot dockerfile or a doc there)

Reply:
@ldemailly since we've moved the iptables work to istio/istio, it would seem that any documentation belongs in those Dockerfiles, not here.  That way, these just become dockerfiles for our own custom envoy deployment and very little more.  WDYT?

Reply:
sounds good, but what's the PR adding them on the other side (sorry if I missed it). I just want to avoid losing the setup/info.

Reply:
@ldemailly I think this is answered by https://github.com/istio/proxy/pull/1131#pullrequestreview-97355637.  Short answer is: I don't think we need another PR.